webpack.config.js

```diff
@@ -167,7 +167,7 @@ const extraScriptConfig = Object.assign({}, baseConfig, {
 		alias: {
 			api: path.resolve(__dirname, 'api'),
 		},
-		extensions: ['.tsx', '.ts', '.js'],
+		extensions: ['.tsx', '.ts', '.js', '.json'],
```

Comment:
The original problem was happening because the library needs to import some JSON files and Webpack didn't resolve them for some reason, so adding `.json` fixes the problem.

Reply:
Right, that makes sense. So does it mean that somewhere in the library code there's a `require('./somefile.json')`?

Reply:
Yes

Reply:
Ok that's good to know. I'll add this to the default config.